docker/Dockerfile.nvidia

```diff
@@ -84,19 +79,6 @@ ENV CUDA_ROOT $HPCSDK_HOME/cuda/bin
 ENV PATH $HPCSDK_HOME/compilers/bin:$HPCSDK_HOME/cuda/bin:$HPCSDK_HOME/comm_libs/mpi/bin:${PATH}
 ENV LD_LIBRARY_PATH $HPCSDK_HOME/cuda/lib:$HPCSDK_HOME/cuda/lib64:$HPCSDK_HOME/compilers/lib:$HPCSDK_HOME/math_libs/lib64:$HPCSDK_HOME/comm_libs/mpi/lib:$HPCSDK_CUPTI/lib64:bitcomp_DIR:${LD_LIBRARY_PATH}
 
-ADD ./requirements.txt /app/requirements.txt
```

Comment:
I'm confused, where are these going?

Reply:
Instead of copying from the host to the image, the Dockerfile clones the latest Devito.
https://github.com/devitocodes/devito/pull/1797/files#diff-712980f1b6ff2592c72da4a45d841c8408275e3b9178043c7db3736d7c30ba68R113

Reply:
I think this is towards the Good direction